src/PresentationalComponents/Tailorings/Tailorings.js

```diff
@@ -7,71 +7,155 @@ import {
   StateViewPart,
 } from 'PresentationalComponents';
 import useTailorings from 'Utilities/hooks/api/useTailorings';
+import useSecurityGuide from 'Utilities/hooks/api/useSecurityGuide';
+
 import OsVersionText from '../TabbedRules/OsVersionText';
 import TailoringTab from './components/TailoringTab';
 import { eventKey } from './helpers';
 
 // TODO Systems count on tabs -> may need API change
 // TODO defaultTab  defaultTab={getDefaultTab(tailorings, defaultTab)}
+
+/**
+ * This component is used to show either the tailorings with rules of a specific policy,
+ * or the rules of a specific security guide and it's rules for a set of minor OS versions
+ *
+ *  @param                        [props]
+ *  @param                        [props.policy]              A policy object from the API
+ *  @param                        [props.securityGuideId]     A security guide ID
+ *  @param                        [props.osMinorVersions]     An array of OS minor versions to show the security guide rules for
+ *  @param                        [props.defaultTab]
+ *  @param                        [props.columns]             An array of RulesTable columns
+ *  @param                        [props.level]
+ *  @param                        [props.ouiaId]
+ *  @param                        [props.onValueOverrideSave] Callback function called when a value of a rule is saved
+ *
+ *  @param                        props.onSelect
+ *  @param                        props.preselected
+ *
+ *  @returns {React.ReactElement}
+ *
+ *  @category Compliance
+ *
+ */
 const Tailorings = ({
   policy,
+  securityGuideId,
+  osMinorVersions,
   defaultTab,
   columns,
   level = 0,
   ouiaId,
   onValueOverrideSave,
+  onSelect,
+  preselected,
   ...rulesTableProps
 }) => {
-  const { data, loading, error } = useTailorings({
+  const {
+    data: tailoringsData,
+    loading: tailoringsLoading,
+    error: tailoringsError,
+  } = useTailorings({
     params: [
       policy.id,
```

Comment:
```suggestion
      policy?.id,
```

in case we don't want to fetch tailorings and don't provide _policy_ prop

Reply:
Changed.